src/interface.rs

```diff
@@ -41,9 +43,16 @@ pub enum MenuMode {
 }
 
 impl MenuMode {
-    fn text(&self) -> &str {
+    fn text(&self, interface: &Interface) -> &str {
         match *self {
-            MenuMode::Normal => "McFly | ESC - Exit | ⏎ - Run | TAB - Edit | F2 - Delete",
+            MenuMode::Normal => match interface.settings.key_scheme {
```

Comment:
Can this just be `self.settings.key_scheme` so that you don't need to pass `interface` in?

Reply:
It seems not. I'm not 100% but I think `self` here is of type `MenuMode`, which has no reference to `settings` because that's on `Interface`.

Reply:
Yea, that makes sense.